data/.schema.yaml

```diff
@@ -91,6 +91,20 @@ properties:
       required:
         - measurements
       properties:
+        attributes:
```

Comment:
This section of the schema declares that we can add an object `attributes` to each participant, but it is not required (because it's not listed in the `required` section just above). Arbitrary attributes can be added, but `sex` and `age` are validated against the schema below.